cypress/integration/supervision/dashboard/reassign-client.spec.js

```diff
@@ -39,42 +39,47 @@ before(function setupAllocatedClient () {
     .then(allocateTheClientToHWTeam);
 });
 
-describe('Reassign client on the dashboard', { tags: ['@supervision', '@supervision-regression', '@xxx'] }, () => {
-  it(
-    'Given I\'m a Case Manager on Supervision and on the client dashboard page' +
-    'Then the Client Dashboard page loads as expected',
-    () => {
-      cy.loginAs('Lay User');
-      cy.visit('/supervision/#/dashboard');
-      cy.get('li[class="tab-container__tab TABS_CLIENTS"]').click();
+describe(
+  'Reassign client on the dashboard',
+  { tags: ['@supervision', '@supervision-core', '@supervision-regression', '@reassign-client-dashboard', '@smoke-journey'] },
+  () => {
+    it(
+      'Given I\'m a Case Manager on Supervision and on the client dashboard page' +
+      'Then the Client Dashboard page loads as expected',
```

Comment:
This will render as one line in the test output without a space between page and Then. If you want to stick with the BDD style naming, a better option would be to use backticks instead of single quotes, as they are multi-line strings. e.g.
```
   it(
      `Given I'm a Case Manager on Supervision and on the client dashboard page
        Then the Client Dashboard page loads as expected`,
```

Alternatively, you can use the context function to provide the Given/When/Then statements: https://dev.to/walmyrlimaesilv/using-the-keywords-given-when-then-with-cypress-but-without-cucumber-118p